campaignion_loqate/campaignion_loqate.module

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations callbacks for this module.
+ */
+
+use \Drupal\manual_direct_debit_uk\AccountDataController;
+
+/**
+ * Implements hook_payment_forms_payment_form_alter().
+ */
+function campaignion_loqate_payment_forms_payment_form_alter(&$form, \Payment $payment) {
+  if (!($payment->method->controller instanceof AccountDataController)) {
+    return;
+  }
+  if (!($key = variable_get_value('campaignion_loqate_api_key'))) {
+    return;
+  }
+  $form['#attributes']['class'][] = 'loqate-validate';
```

Comment:
This class isn’t used anywhere anymore. Do you want to keep it anyway?

Reply:
I’ve removed it.